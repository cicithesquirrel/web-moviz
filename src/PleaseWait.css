#wait {
    text-align: center;
    font-size: x-large;
    font-style: italic;
    color: Â #666;
    margin: 1em 0;
}