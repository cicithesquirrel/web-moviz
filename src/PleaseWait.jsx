import React from 'react';
import "./PleaseWait.css";

const PleaseWait = ({crumbs}) => (
    <div id="wait"><i className="fa fa-spinner fa-pulse fa-fw"></i> Please wait...</div>
);

export default PleaseWait;