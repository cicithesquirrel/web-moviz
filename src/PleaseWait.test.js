import React from 'react';
import PleaseWait from './PleaseWait';
import renderer from 'react-test-renderer';

test('PleaseWait', () => {
  const component = renderer.create(
    <PleaseWait />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});