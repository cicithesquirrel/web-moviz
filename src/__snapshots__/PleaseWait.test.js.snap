exports[`test PleaseWait 1`] = `
<div
  className="Generic-message">
  <i
    className="fa fa-spinner fa-pulse fa-fw" />
  Â  Please wait...
</div>
`;
